tools/geojson_simplifier/download.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Downloads and saves GeoJson map files from DataCommons.
+    Typical usage:
+    python3 download.py
+"""
+
+import datacommons as dc
+import geojson
+
+
+# TODO(fpernice-google): Support downloading more than just US states.
+class GeojsonDownloader:
+    """Downloads desired GeoJSON files from the DataCommons Knowledge Graph.
+
+    Attributes:
+        geojsons: A dictionary that maps each queried area to another
+                  dictionary containing the GeoJSON coordinate information. An
+                  example of this is the following
+
+                {   # Multipolygon of the state of Alabama (fake).
+                    "geoId/01": [{
+                        "type": "MultiPolygon",
+
+                        # Set of individual Polygons that compose it.
+                        "coordinates": [
+                            # Polygon 1
+                            [[ [1.5, 12.4], [5.3, 45.2], [1.1, 3.5],
+                                                            [1.5, 12.4] ]],
+                            # Polygon 2
+                            [[ [1, 2], [3, 4], [5, 6], [2, -1], [1, 2] ]],
+                            # Polygon 3
+                            [[ [53, 23], [65, 2], [31, 12], [53, 23] ]]
+                        ]
+                    }],
+                    # Polygon of the state of Wyoming (fake).
+                    # Since Wyoming is a single chunk of land, its type
+                    # is Polygon instead of Multipolygon.
+                    "geoId/17": [{
+                        "type": "Polygon",
+                        "coordinates": [
+                            # Polygon 1
+                            [[ [1.5, 12.4], [5.3, 45.2], [1.1, 3.5],
+                                                            [1.5, 12.4] ]]
+                        ]
+                    }]
+
+                }
+    """
+    LEVEL_MAP = {
+        "Country": "AdministrativeArea1",
+        "AdministrativeArea1": "AdministrativeArea2",
+        "AdministrativeArea2": "City"
+    }
+
+    def __init__(self):
+        dc.set_api_key('dev')
```

Comment:
This isn't necessary anymore! Besides, dev isn't a correct key.

Reply:
Getting the following error when trying to run without this line:
```
Traceback (most recent call last):
  File "download.py", line 118, in <module>
    loader.download_data()
  File "download.py", line 86, in download_data
    geolevel_below = dc.get_property_values([place],
  File "/home/fpernice/.local/lib/python3.8/site-packages/datacommons/core.py", line 174, in get_property_values
    payload = utils._send_request(url, req_json=req_json)
  File "/home/fpernice/.local/lib/python3.8/site-packages/datacommons/utils.py", line 94, in _send_request
    raise ValueError(
ValueError: Request error: Must set an API key before using the API! You can call datacommons.set_api_key or assign the key to an environment variable named DC_API_KEY
```

Reply:
It seems like putting in any API key works though... Not sure why that is.